### ISSUE TITLE
Document m.cap()

### ISSUE THREAD

**Issue by joakim** [pull-stream/pull-many#3]
First of all, thanks for all the great pull-stream plugins! I'm loving the small code and ease of use of these modules, this is what streams should be like.

There's one thing that's bugging me though, and that's the lack of documentation. You have some very good explanations and examples on the main `pull-stream` module, but some of the modules in the ecosystem aren't that well documented. In this case, I spent too many hours scratching my head before finding out I had to call `m.cap()`when adding streams using `m.add()`. A one-liner in the readme would've prevented that.

I don't mean to sound unappreciative, I'm actually very grateful for these modules and hope they will be discovered and used by more people. But documentation is important for adoption. Maybe I could contribute with documentation as I go exploring the pull stream ecosystem?

**Comment by dominictarr** [comment]
Yes you are absolutely right. when I started pull streams, I was mostly doing it for fun, and I didn't care about adoption. I think this phase was important because it allowed be to become very confidant that pull-streams where the _right idea_. Later when I used pull-streams in a big project ([secure-scuttlebutt](https://github.com/ssbc/scuttlebot) ) and adoption started to become important. Adoption has picked up recently, and most of the credit belongs to @ahdinosaur. partly what he did was encourage me to write some, and also he created the [pull-stream doc site](http://pull-stream.github.io/) and generally encouraged the community to grow.

I've invited you to the pull-stream org, that means you can easily add docs to any pull-stream repo. It's best to use pull requests, because that will send an email to the other members, and generally give the community chances to interact. If you know a change is correct and just want to merge it, make the PR and then merge it right away.

I've invited you as an "owner" too, that means you can invite other people too! if you see someone doing really good pull-stream work, please invite them!

we should probably figure out a way to efficiently add everyone as npm owners too, because that is still a permodule basis...
